packages/socket-mode/README.md

```diff
@@ -0,0 +1,245 @@
+# Slack Socket Mode
```

Comment:
Oops I included all of my edits in the non-markdown version sorry :(

Reply:
oh, either file works fine.